### PR TITLE
Gun gifts

### PR DESCRIPTION
**Pull request by mikolaj007**
## Changelog
:cl:
add: Wizard federation has decided to summon some guns and package them into christmas gifts
add: a new type of gift has been added for adminbus
/:cl:

## About The Pull Request
Adds a gift type that gives you the effect of a summon guns spell

## Why It's Good For The Game
in case someone wants to make gun santa or have a christmas tree that distributes guns

### CODE REVIEW

**Comment by Majick1** [comment]
+1 merry Christmas

File: code/game/objects/items/gift.dm [hunk 1]
```diff
@@ -111,3 +111,27 @@ GLOBAL_LIST_EMPTY(possible_gifts)
...
 	return gift_type
+
+
+
+/obj/item/a_gift/gun
+	name = "christmas gift"
+	desc = "It's vaguely gun shaped"
+
+/obj/item/a_gift/gun/get_gift_type()
+
```
Comment (Manly-Dwarf) [inline code/game/objects/items/gift.dm]: What's the point of this newline?

File: code/game/objects/items/gift.dm [hunk 2]
```diff
@@ -111,3 +111,27 @@ GLOBAL_LIST_EMPTY(possible_gifts)
...
+	var/gun_type = pick(GLOB.summoned_guns)
+	return gun_type
+/obj/item/a_gift/attack_self(mob/M)
```
Comment (Manly-Dwarf) [inline code/game/objects/items/gift.dm:125]: Remove the newline above and put it here instead, so the attack_self and return are separated.

**Review by ghost — APPROVED** [review]
looks good to me

**Comment by AnthonyJNgo** [comment]
Merry Christmas

**Pull request merged**